migrations/versions/3d3fb04d1a98_.py

```diff
@@ -0,0 +1,36 @@
+"""empty message
```

Comment:
We're creating more and more migrations, so I think it's worth running `python application.py db migrate` with `-m MESSAGE` in the future. It will add a `_<message>` to the migration file name, which makes it easier to tell them apart.

I think we can rename these files safely too by adding something after `_` without breaking migration versions.